samples/simulation/ising/README.md

```diff
@@ -0,0 +1,110 @@
+---
+page_type: sample
+languages:
+- qsharp
+products:
+- qdk
+description: "These samples demonstrate how to use Q# to simulate the Ising model."
+urlFragment: ising
+---
+
+# Ising Model Simulation Samples
+
+These samples demonstrate how to use Q# to simulate the Ising model.
+
+- **[Simple Ising](#simple-ising-sample)**: This sample walks through constructing the time-evolution operator for the Ising model.
+- **[Ising Generators](#ising-generators-sample)**: This sample describes how Hamiltonians may be represented using Microsoft.Quantum.Canon functions.
+- **[Adiabatic Ising Evolution](#adiabatic-ising-evolution-sample)**: This sample converts a representation of a Hamiltonian using library data types into unitary time-evolution by the Hamiltonian on qubits.
+- **[Ising Phase Estimation](#ising-phase-estimation-sample)**: This sample adiabatically prepares the ground state of the Ising model Hamiltonian, and then perform phase estimation to obtain an estimate of the ground state energy.
+- **[Ising Trotter Evolution](#ising-trotter-sample)**: This sample walks through constructing the time-evolution operator for the Ising model using the Trotterization library feature.
+
+## Manifest
+
+- [IsingSamples.csproj](./IsingSamples.csproj): Main Q# project for all of the Ising samples.
+
+## Running the Samples
+
+### In Visual Studio Code or the Command Line
+
+At a terminal, run the following command, replacing `<entry point>` with the name of the sample's entry point:
+
+```dotnetcli
+dotnet run simulate <entry point>
+```
+
+### In Visual Studio 2019
+
+Open the `simulation.sln` solution in Visual Studio and set the .csproj file in the manifest as the startup project.
+Set the command-line arguments to `simulate <entry point>`, replacing `<entry point>` with the name of the sample's entry point.
+Press Start in Visual Studio to run the sample.
+
+## Simple Ising Sample
+
+**Entry Point:** `Microsoft.Quantum.Samples.Ising.RunSimple`
+
+This sample walks through constructing the time-evolution operator for the Ising model manually.
+This time-evolution operator is applied to adiabatically prepare the ground state of the Ising model.
+The net magnetization is then measured.
+
+### Manifest
```

Comment:
As noted in my earlier reply, I think we can turn off this warning by using the `siblings_only` option to MD024. I agree with you, there's no problem with non-sibling headings being duplicated.